### PR TITLE
MultiWhisper

### PR DESCRIPTION
**Pull request by funkydemir66**
MultiWhisper for G Earth

### CODE REVIEW

File: store/extensions/MultiWhisper/extension.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "title": "Multi Whisper",
```
Comment (Alynva) [inline store/extensions/MultiWhisper/extension.json]: Remove the space in the title, or add it in the folder's name

**Review by Alynva — CHANGES_REQUESTED** [review]

**Comment by funkydemir66** [comment]
okey finished

**Review by Alynva — APPROVED** [review]

File: store/extensions/MultiWhisper/extension.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "stable": true,
+    
+    "framework": {
+      "name": "G-Python",
+      "version": "1.6"
+    },
+
+    "language": "Python",
+    "commands": {
+      "default": ["python", "ModTools.py", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"]
```
Comment (WiredSpast) [inline store/extensions/MultiWhisper/extension.json]: This doesn't seem right

**Review by WiredSpast — CHANGES_REQUESTED** [review]

File: store/extensions/MultiWhisper/extension.json (hunk 2 above)
Comment (Alynva) [reply store/extensions/MultiWhisper/extension.json]: Yeah, it should be `MultiWhisper.py`

**Review by Alynva — CHANGES_REQUESTED** [review]

File: store/extensions/MultiWhisper/extension.json (hunk 2 above)
Comment (funkydemir66) [reply store/extensions/MultiWhisper/extension.json]: its okey

Comment (funkydemir66) [reply store/extensions/MultiWhisper/extension.json]: ty for helps

**Review by WiredSpast — APPROVED** [review]

**Review by Alynva — APPROVED** [review]

**Comment by funkydemir66** [comment]
okey?

**Comment by funkydemir66** [comment]
wh

File: store/extensions/RollerWalk/extension.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+    "title": "Multi Whisper",
```
Comment (Alynva) [inline store/extensions/RollerWalk/extension.json]: Your extension.json of the RollerWalk isn't right.

**Review by Alynva — CHANGES_REQUESTED** [review]
Please, try to not submit two extensions in the same PR.

**Comment by WiredSpast** [comment]
> Please, try to not submit two extensions in the same PR.

That actually doesn't matter

File: store/extensions/RollerWalk/extension.json (hunk 3 above)
Comment (funkydemir66) [reply store/extensions/RollerWalk/extension.json]: yes i will do it there

**Comment by Alynva** [comment]
> > Please, try to not submit two extensions in the same PR.
> 
> That actually doesn't matter

For the tests work I know that doesn't matter, I just think that helps to keep the conversation centralized

**Comment by funkydemir66** [comment]
all ok i think

**Pull request merged**